src/notifications/webhookManager.ts

```diff
@@ -0,0 +1,60 @@
+import { SpvWalletClient } from '../client';
+import { Events } from '../types';
+import { EventsMap } from './eventsMap';
+import { EventHandler, RawEvent, WebhookOptions } from './interface';
+import { Request, Response } from 'express';
+
+export class WebhookManager {
+  public url: string;
+  public options: WebhookOptions;
+  public subscriber: SpvWalletClient;
+  public handlers: EventsMap;
+
+  constructor(subscriber: SpvWalletClient, url: string, options: WebhookOptions = {}) {
+    this.url = url;
+    this.subscriber = subscriber;
+    this.options = {
+      tokenValue: options.tokenValue || '',
+      tokenHeader: options.tokenHeader || '',
+    };
+    this.handlers = new EventsMap();
+  }
+
+  subscribe() {
+    return this.subscriber.AdminSubscribeWebhook(this.url, this.options.tokenHeader!, this.options.tokenValue!);
+  }
+
+  unsubscribe() {
+    return this.subscriber.AdminDeleteWebhook(this.url);
+  }
+
+  async httpHandler(req: Request, res: Response) {
```

Comment:
Our `js-client` should work also on `mobile` and `web` so adding `express` as a dependency is problematic. 

Also, what if a `js-client-user` wants to use other framework than `express`.
Consider some interface which user have to provide to handle the "request" stuff. 
This way, users can implement the interface with the framework of their choice.

Reply:
added more generic way to handle the webhook notifier